### PR TITLE
Cypress test. Update "Multiple users. Assign task, job."

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Test updated to check the fix #3343 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@ActiveChooN , could you please review?

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js [hunk 1]
```diff
@@ -133,5 +158,23 @@ context('Multiple users. Assign task, job.', () => {
...
+        it('First user login. Getting authKey.', () => {
+            cy.visit('/');
+            cy.intercept('POST', '/api/v1/auth/login').as('login');
+            cy.login();
+            cy.wait('@login').then((response) => {
+                authKey = response['response']['body']['key'];
+            });
+        });
+
+        it('Deactivate the second users (task assigned). Trying to open the task. Should be succefull.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js:171]:
```suggestion
        it('Deactivate the second user (task assigned). Trying to open the task. Should be successful.', () => {
```

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js [hunk 2]
```diff
@@ -133,5 +158,23 @@ context('Multiple users. Assign task, job.', () => {
...
+            changeCheckUserStatusOpenTask(secondUserName);
+            cy.goToTaskList();
+        });
+
+        it('Deactivate the third users (job assigned). Trying to open the task. Should be succefull.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js:176]:
```suggestion
        it('Deactivate the third user (job assigned). Trying to open the task. Should be successful.', () => {
```

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js [hunk 3]
```diff
@@ -133,5 +158,23 @@ context('Multiple users. Assign task, job.', () => {
...
+            changeCheckUserStatusOpenTask(secondUserName);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js:177]: Maybe thirdUserName?

File: tests/cypress/support/commands.js [hunk 4]
```diff
@@ -83,6 +83,55 @@ Cypress.Commands.add('deletingRegisteredUsers', (accountToDelete) => {
...
+Cypress.Commands.add('changeUserActiveStatus', (authKey, accountsToChangeActiveStatus, isActive) => {
+    let status;
+    cy.request({
+        url: '/api/v1/users?page_size=all',
+        headers: {
+            Authorization: `Token ${authKey}`,
+        },
+    }).then((response) => {
+        const responceResult = response['body']['results'];
+        for (const user of responceResult) {
```
Comment (bsekachev) [inline tests/cypress/support/commands.js:95]: what do you think about this code?

```js
responseResult.filter((user) => accountsToChangeActiveStatus.includes(user['username']))
.forEach(() => {
    cy.request({
        method: 'PATCH',
        url: `/api/v1/users/${userId}`,
        headers: {
              Authorization: `Token ${authKey}`,
        },
        body: {
               is_active: isActive,
         }
     });
})

```

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js]: Applied locally

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js]: Applied locally

File: tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js]: Yes, of course. Thanks.

File: tests/cypress/support/commands.js (hunk 4 above)
Comment (dvkruchinin) [reply tests/cypress/support/commands.js]: Unfortunately, in this case I get error ```responseResult.filter is not a function```. But I tried to simplify this code somewhat.

**Pull request merged**